pkgs/by-name/le/lessc/wrapper/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  stdenv,
+  makeWrapper,
+  lessc,
+  plugins ? [ ],
+}:
+
+let
+  makeNodePath = paths: lib.makeSearchPath "/lib/node_modules" paths;
+in
+stdenv.mkDerivation {
+  pname = "lessc-with-plugins";
+  nativeBuildInputs = [ makeWrapper ];
+  buildPhase = ''
+    mkdir -p $out/bin
+
+    makeWrapper "${lib.getExe lessc}" "$out/bin/${lessc.meta.mainProgram}" \
```

Comment:
```suggestion
    makeWrapper "${lib.getExe lessc}" ${lib.getExe lessc} \
```

Reply:
I don't understand this suggestion, `${lib.getExe lessc}` is not writable :confused:

Reply:
ping @SuperSandro2000